src/app/app.ts

```diff
@@ -150,6 +161,27 @@ class AppImpl implements App {
         return newOKCallResponseWithMarkdown(msg);
     }
 
+    validateSubNameIsUnique = (subName: string): boolean => {
+        const state = this.call.state;
```

Comment:
What's the reasoning for using `state`? Is it so we don't re-compute the subscriptions across form interactions?

Reply:
Yes, I wanted to minimize the number of Zendesk calls.  I've got the subs available after the first opening of the modal so I just stored it in state and pass it around.